src/npms-package-info.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const moment = require('moment');
+const got = require('got');
+const pMap = require('p-map');
+const log = require('./logger');
+
+module.exports = list => {
+  log.info('npmsInfo: Fetching info for %s packages', list.length);
+
+  return pMap(list, fetchInfos, {concurrency: 2})
+    .then(pkgs => {
+      pkgs = pkgs.filter(pkg => Boolean(pkg));
+      log.info('npmInfo: Fetched info for %s valid packages', pkgs.length);
+
+      return pkgs;
+    })
+    .catch(err => {
+      console.error(err);
```

Comment:
This should either use the logger or preferably just not `.catch()` and let the error bubble up to the next `.catch()`.